OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -156,6 +134,23 @@ public void InvokeCommand(string name, string arg)
 			}
 		}
 
+		void GiveCash(bool toAll, string arg)
+		{
+			string orderString = "";
```

Comment:
This can be shortened to
`var orderString = toAll ? "DevGiveCashAll" : "DevGiveCash";`